### PR TITLE
Django client gen

### PR DESCRIPTION
**Pull request by lucaslcode**

### CODE REVIEW

**Comment by lucaslcode** [comment]
This adds a yarn command `generate-api` to generate a type-safe api from the openapi schema at https://backend-v2-dev.couchers.dev/schema/

The changes in `newClient.ts` and `user.ts` show how we might use this in the service layer. (However `loginCreate` response type isn't correctly defined in the schema at the moment)

File: service/user.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ import {
 } from "proto/api_pb";
 import { AuthReq, CompleteTokenLoginReq } from "proto/auth_pb";
 import client from "service/client";
+import newClient from "service/newClient";
```
Comment (guillermodlpa) [inline service/user.ts:17]: is there a way to avoid calling this file "newClient"? Maybe "restClient" or some other attribute to differentiate it?

**Review by guillermodlpa — COMMENTED** [review]
This is so cool! So updating the schema in the frontend should be rather easy this way, without much back and forth, and not needing to define a lot of boilerplate to create the client and all the payload types, right? 🎉

**Comment by aapeliv** [comment]
Django never happened.

**Pull request closed without merging**